style/amenity-points.mss

```diff
@@ -1365,16 +1365,19 @@
     marker-fill: @man-made-icon;
   }
 
-  [feature = 'natural_peak'][zoom >= 11] {
-    marker-file: url('symbols/natural/peak.svg');
+  [feature = 'natural_peak'][zoom >= 10],
+  [feature = 'natural_volcano'][zoom >= 10] {
     marker-fill: @landform-color;
     marker-clip: false;
-  }
-
-  [feature = 'natural_volcano'][zoom >= 11] {
-    marker-file: url('symbols/natural/peak.svg');
-    marker-fill: #d40000;
-    marker-clip: false;
+    [feature = 'natural_peak'] {
+      marker-file: url('symbols/natural/peak.svg');    
+    }
+    [feature = 'natural_volcano'] {
+      marker-file: url('symbols/natural/volcano.svg');    
+    }
+    [zoom = 10] {
+      marker-width: 4;
+    }
   }
```

Comment:
I would rather split this up into two blocks than put both peaks and volcanos ihn the same one. 

e.g.
```
[feature = 'natural_peak'][zoom >= 10] {
...
}
[feature = 'natural_volcano'][zoom >= 10] {
...
}
```

Reply:
done!